src/App.css

```diff
@@ -0,0 +1,3 @@
+iframe {
+  display: none;
+}/*# sourceMappingURL=App.css.map */
```

Comment:
Consider fixing the error here 
<img width="748" alt="Screenshot 2023-11-05 at 19 10 37" src="https://github.com/mate-academy/react_tabs-js/assets/135232001/0fb57243-1b0a-4e38-b8ce-d2f089f67986">